### DISCUSSION TITLE
Plotting material properties

### DISCUSSION THREAD

**Discussion by Khaksarkn** [idaholab/moose#25334] [Q&A Tools] [answered]
Hi,

I was wondering if there is any way to plot material properties in the exodus output. Say I have defined a material property X which takes different values at _qp 's, and I want to see what the distribution of X is at each time step. 

is there a way to get the same plot we get for our main variables (on _qp 's), for a material property?

Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Yes it s very easy. You use the outputs parameter of the material block and say which 
Not on qps though. You ll get a cell-average

Guillaume
